templates/trivy/trivy-sts.yaml

```diff
@@ -30,20 +30,57 @@ spec:
 {{ toYaml .Values.trivy.podAnnotations | indent 8 }}
 {{- end }}
     spec:
+      automountServiceAccountToken: false
 {{- with .Values.imagePullSecrets }}
       imagePullSecrets:
         {{- toYaml . | nindent 8 }}
 {{- end }}
 {{- if .Values.trivy.serviceAccountName }}
       serviceAccountName: {{ .Values.trivy.serviceAccountName }}
 {{- end }}
-      securityContext:
-        runAsNonRoot: true
-        runAsUser: 10000
-        fsGroup: 10000
-      automountServiceAccountToken: false
+      initContainers:
+        - name: "change-directory-owner"
```

Comment:
I don't think you need this init container. The `securityContext.fsGroup: 10000` setting is already taking care of it.

Reply:
Yeah, I'm not sure about this. We had it in our entrypoint script and without this chown I'm getting permission problems in the log...